modules/ROOT/pages/microsoft/microsoft-dynamics-365-connector.adoc

```diff
@@ -86,19 +94,19 @@ timing out. Zero (0) means wait forever.
 . Click Test Connection to ensure that your credentials are accepted at the server endpoint.
```

Comment:
UI should be bolded Click **Test Connection**.... click **Enable DataSense**...etc

Reply:
Fixed